app/javascript/packs/articleModerationTools.js

```diff
@@ -12,7 +12,7 @@ getCsrfToken().then(() => {
       const { initializeActionsPanel } = await import(
         '../actionsPanel/initializeActionsPanelToggle'
       );
-      const { initializeFlagUserModal } = await import('./flagUserModal');
+      // const { initializeFlagUserModal } = await import('./flagUserModal');
```

Comment:
The comments in this file are because we are not preloading `flagUserModal` for use in the Mod Center, for now. Please refer to [this Slack conversation](https://forem-team.slack.com/archives/C03CHBBMTJP/p1658500957993839) for more context.